client/src/components/ArrowsRenderer/index.tsx

```diff
@@ -178,46 +186,74 @@ const arrowsUpdate = (update: any) => {
   return update;
 };
 
+const feetHelpers = {
+  firstStroke(d: [string, FeetData]) {
+    const { verdicts } = d[1].accessPointMeta;
+    if (verdicts.has(Verdict.Forwarded) && verdicts.has(Verdict.Dropped)) {
+      return undefined;
+    } else if (verdicts.has(Verdict.Dropped)) {
+      return colors.feetDroppedStroke;
+    }
+    return colors.feetForwardedStroke;
+  },
+  secondStroke(d: [string, FeetData]) {
+    const { verdicts } = d[1].accessPointMeta;
+    if (verdicts.has(Verdict.Forwarded) && verdicts.has(Verdict.Dropped)) {
+      return colors.feetDroppedStroke;
+    }
+    return undefined;
+  },
+  secondStrokeDasharray(d: [string, FeetData]) {
+    return d[1].accessPointMeta.verdicts.size > 1 ? '5 4' : undefined;
+  },
+};
+
 const feetsEnter = (enter: any) => {
-  const feetGroup = enter.append('g').attr('class', (d: any) => d[0]);
+  const feetGroup = enter
+    .append('g')
+    .attr('class', (d: [string, FeetData]) => d[0]);
 
   feetGroup
     .append('line')
-    .attr('class', 'outer')
-    .attr('x1', (d: any) => d[1][0].x)
-    .attr('y1', (d: any) => d[1][0].y)
-    .attr('x2', (d: any) => d[1][1].x)
-    .attr('y2', (d: any) => d[1][1].y)
-    .attr('stroke', colors.feetOuterStroke)
-    .attr('stroke-width', sizes.feetOuterWidth);
+    .attr('class', 'feet-first')
```

Comment:
Something is wrong here:
Previous code renders two layers for each feet stick: the lower one has brighter color to make it visible when overlapped with dark thick arrow, the upper one is feet stick itself. And they had obvious names: outer and inner due to that. Strongly recommend to revert this code and adapt new values to that layers.